package.json

```diff
@@ -1,12 +1,7 @@
 {
   "name": "node-emoji",
   "version": "2.1.0",
-  "description": "Friendly emoji lookups and parsing utilities for Node.js",
-  "author": "Daniel Bugl <me@omnidan.net>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/omnidan/node-emoji.git"
-  },
+  "description": "Friendly emoji lookups and parsing utilities for Node.js. 😏",
```

Comment:
small thing but maybe we can use the 💖 emoji here, I feel it is more friendly and universal than 😏 which could be misunderstood.

Reply:
I love it! 💖 is my favorite one.